docs/data/data-strategy.md

```diff
@@ -0,0 +1,87 @@
+# Data Strategy
+
+Mission: Help Thoth Tech Make Better Decisions, Faster.
+
+At Thoth Tech, we endeavor to be user-centered. Users will be the center of our operations including
+how we develop and support our products. To support that mission, we would like to have strong data
+insights.
+
+We are creating a Data Platform, a single unified data and analytics stack. The Data Platform will
+power cross-functional reporting and analysis, and in general, allow all team members to make better
+decisions with trusted data. Over time, the Data Platform will further enhance Thoth Tech's
+analytics capabilities with features such as data products - enriched and aggregated data integrated
+into Thoth Tech products for use by our users. This acceleration happens through the development of
+"Data Flywheels".
+
+<details>
+<summary>What is a flywheel strategy?</summary>
+
+[A flywheel strategy](https://www.ejorgenson.com/blog/flywheel-effect) has positive feedback loops
+that build momentum, increasing the payoff of incremental effort.
+
+</details>
+
+## Data Flywheels
+
+We call this User and Product Intelligence Flywheel. It is focused on improving the User Experience
+and encompasses the data and analytics involved in the user-product interactions, user use cases,
+product development and most aspects of the user journey.
+
+In the open core flywheel, more features drive more users which in turn drive more contributions
+which lead to more users.
+
+```mermaid
```

Comment:
This is super cool!
Please add this into the Markdown Guide! 🙏

Reply:
Totally, this is a pretty new feature on GitHub.

Reply:
Raised an issue to follow up https://github.com/thoth-tech/handbook/issues/65